nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/CreateHadoopSequenceFile.java

```diff
@@ -88,7 +92,7 @@
     }
     // Optional Properties.
     static final PropertyDescriptor COMPRESSION_TYPE = new PropertyDescriptor.Builder()
-            .name("compression type")
+            .name("Compression type")
```

Comment:
Isn't this is a breaking change? I see the typo but better add the correct spelling as displayName